docs/versions.rst

```diff
@@ -3,6 +3,7 @@ Available Documentation for AutoGluon-Cloud
 
 Web-based documentation is available for versions listed below:
 
-- `AutoGluon 0.1.1 (dev) documentation <https://auto.gluon.ai/cloud/dev/index.html>`_
-- `AutoGluon 0.1.0 (stable) documentation <https://auto.gluon.ai/cloud/stable/index.html>`_
+- `AutoGluon 0.2.1 (dev) documentation <https://auto.gluon.ai/cloud/dev/index.html>`_
+- `AutoGluon 0.2.0 (stable) documentation <https://auto.gluon.ai/cloud/stable/index.html>`_
+- `AutoGluon 0.2.0 documentation <https://auto.gluon.ai/cloud/0.2.0/index.html>`_
```

Comment:
Do you want `0.2.0` to have 2 lines here? Its not a big deal but it is different than how we do it in AutoGluon

Reply:
Huh, I remember I was following AutoGluon. NVM, I'll update it